pkg/dpll/dpll.go

```diff
@@ -649,20 +650,24 @@ func (d *DpllConfig) stateDecision() {
 		} else { // what happens if source is lost and DPLL is locked? goto holdover?
 			glog.Infof("dpll is locked, offset is out of range, state is FREERUN(%s)", d.iface)
 			d.state = event.PTP_FREERUN
+			d.phaseOffset = FaultyPhaseOffset
```

Comment:
we don't see offset here , because DPLL is in locked state and the netlink event should have updated the offset